gradle.properties

```diff
@@ -1,4 +1,30 @@
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx4g -XX:MaxMetaspaceSize=2g
 org.gradle.parallel=true
 org.gradle.caching=true
-org.gradle.configuration-cache=true
\ No newline at end of file
+org.gradle.configuration-cache=true
+
+
+GROUP=com.gradleup.shadow
```

Comment:
Maybe `com.gradleup.plugins`? Started the discussion on Slack

Reply:
Borrowed from https://github.com/GradleUp/nmcp/blob/b6325eac2ca6111a8d22a26be401d021fda90003/build.gradle.kts#L23.

Reply:
I like `com.gradle.shadow`

It allows to easily make a major version bumps if needed (see https://jakewharton.com/java-interoperability-policy-for-major-version-updates/). Also groups all the artifacts together and is short enough.